status/flowcell.py

```diff
@@ -220,7 +220,12 @@ def fetch_ont_run_stats(view_all, view_project, run_name):
 
     # Try to find project name. ID string should be present in MinKNOW field "experiment name" by convention
     query = re.compile("(p|P)\d{5}")
```

Comment:
Might as well have 5 or 6 digits for future proofing?

Reply:
[93b885a](https://github.com/SciLifeLab/genomics-status/pull/740/commits/93b885adb1c6aa6b7eccb8759d5b8da9a4745a6d)